### PR TITLE
Use relative date for reset stale workers

### PR DESCRIPTION
**Pull request by niravi**

### CODE REVIEW

File: scripts/reset_stale_workers.py [hunk 1]
```diff
@@ -85,14 +85,20 @@ def main(arguments):
...
     try:
-        local_cutoff_date = pendulum.from_format(arguments.cutoff_date, "MM/DD/YYYY hh:mm:ss", tz=args.timezone, formatter='alternative')
-    except Exception as e:
-        logger.info(e)
-        logger.info("Invalid date format. Please check documentation and try again")
-        sys.exit(0)
-    utc_dt = local_cutoff_date.in_tz('UTC')
+        delta = int(args.cutoff_date)
+        utc_dt = pendulum.now().subtract(minutes=delta).in_tz('UTC')
+    except Exception:
```
Comment (apulverizer) [inline scripts/reset_stale_workers.py:93]: Might want to narrow the exception here to some Pendulum exception.

**Review by apulverizer — APPROVED** [review]
You could also look at using an interval query as defined in this blog:
https://www.esri.com/arcgis-blog/products/api-rest/data-management/querying-feature-services-date-time-queries/

File: scripts/reset_stale_workers.py (hunk 1 above)
Comment (niravi) [reply scripts/reset_stale_workers.py:93]: I also need to check if the cutoff_date is an int here

**Comment by niravi** [comment]
@apulverizer I want to reuse the same where clause whether or not cutoff_date is a formatted string or an integer for a relative date, and since the situation where the parameter is a formatted string isn't as conducive to an interval query, going to leave it as is

**Pull request merged**